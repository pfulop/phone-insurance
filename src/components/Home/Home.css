.intro {
  font-size: 24px;
  text-align: center;
  padding: 16px;
  flex: 100%;
}

.ready {
  font-size: 32px;
  color: #28112b;
}

.startClaim {
  background-color: #4e8098;
  text-decoration: none;
  text-transform: none;
  color: white;
  padding: 4px 16px;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
  transition: 0.2s;
}

.startClaim:hover {
  box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0.2);
}
