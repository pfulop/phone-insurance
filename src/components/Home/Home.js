import React from 'react';
import { Link } from 'react-router-dom';
import gCss from '../../index.css';
import css from './Home.css';

export default () => (
  <div className={gCss.card}>
    <div className={css.intro}>Don't panic we have got you covered.</div>
    <div className={css.intro}>
      Fill out the form and get your compensation as fast as possible.
    </div>
    <div className={css.intro}>Ready?</div>
    <div className={css.intro}>
      <Link to="/start-claim" className={css.startClaim}>
        Let's start
      </Link>
    </div>
  </div>
);
