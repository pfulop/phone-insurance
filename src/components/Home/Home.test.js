import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';

describe('Home', () => {
  it('should be defined', () => {
    expect(Home).toBeDefined();
  });
  it('should render correctly', () => {
    const tree = shallow(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
