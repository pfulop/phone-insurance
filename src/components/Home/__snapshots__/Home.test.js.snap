// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className={undefined}
>
          Don't panic we have got you covered.
</div>,
        <div
          className={undefined}
>
          Fill out the form and get your compensation as fast as possible.
</div>,
        <div
          className={undefined}
>
          Ready?
</div>,
        <div
          className={undefined}
>
          <Link
                    className={undefined}
                    replace={false}
                    to="/start-claim"
          >
                    Let's start
          </Link>
</div>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Don't panic we have got you covered.",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Don't panic we have got you covered.",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Fill out the form and get your compensation as fast as possible.",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Fill out the form and get your compensation as fast as possible.",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Ready?",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Ready?",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className={undefined}
            replace={false}
            to="/start-claim"
>
            Let's start
</Link>,
          "className": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Let's start",
            "className": undefined,
            "replace": false,
            "to": "/start-claim",
          },
          "ref": null,
          "rendered": "Let's start",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className={undefined}
>
            Don't panic we have got you covered.
</div>,
          <div
            className={undefined}
>
            Fill out the form and get your compensation as fast as possible.
</div>,
          <div
            className={undefined}
>
            Ready?
</div>,
          <div
            className={undefined}
>
            <Link
                        className={undefined}
                        replace={false}
                        to="/start-claim"
            >
                        Let's start
            </Link>
</div>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Don't panic we have got you covered.",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Don't panic we have got you covered.",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Fill out the form and get your compensation as fast as possible.",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Fill out the form and get your compensation as fast as possible.",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Ready?",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Ready?",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className={undefined}
              replace={false}
              to="/start-claim"
>
              Let's start
</Link>,
            "className": undefined,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Let's start",
              "className": undefined,
              "replace": false,
              "to": "/start-claim",
            },
            "ref": null,
            "rendered": "Let's start",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
